firestore/google/cloud/firestore_v1/client.py

```diff
@@ -51,6 +53,7 @@
 _ACTIVE_TXN = "There is already an active transaction."
 _INACTIVE_TXN = "There is no active transaction."
 _CLIENT_INFO = client_info.ClientInfo(client_library_version=__version__)
+_FIRESTORE_EMULATOR_HOST = "FIRESTORE_EMULATOR_HOST"
```

Comment:
Up until now, we have exposed these variables in `google.cloud.environment_vars`:

https://github.com/googleapis/google-cloud-python/blob/ced5858d1f656116015c146454275a4045418e44/core/google/cloud/environment_vars.py#L24-L31

I'm not sure it is the Right Thing(TM), but it is precedent.

Reply:
@tseaver. I saw that and made the active decision I don't like putting those constants there. I think having to release a new core for this seemed a bit silly? I can't imagine those are really used all that broadly? It seemed to me putting a constant I only need in a single client in a shared component will just complicate this.

@busunkim96 do you have thoughts on this? how does this relate to repo-split? I agree with @tseaver consistency has some value so it may be worth deciding now on our direction here?

Reply:
+1 for consistency, see #4840 .

Reply:
Maybe the Right Thing(TM) is to split up the `core/google/cloud/environment_vars.py` module, moving the API-specific values into their own packages.  The advantage of the module is that it makes documenting them easier.

Reply:
No strong feelings here -- I would prefer we move them out to the individual APIs eventually. :woman_shrugging:

Reply:
I am going to leave this like this. I think we may want to push them into the individual packages. Rather than remove from core we may just duplicate though. The end goal should be removing the dependency on core altogether, rather than to start deprecating parts?